### PR TITLE
Kvm support

### PR DESCRIPTION
**Pull request by julialongtin**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

_Briefly describe the issue you have solved or implemented with this pull request. If the PR contains multiple issues, use a bullet list._

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by sysvinit** [comment]
Is there any reasons that tools like `sudo`, `ip`, and `brctl` are accessed indirectly (through shell variables containing hardcoded absolute paths) rather than relying on the operator to have `PATH` set correctly in the environment?

**Comment by stephen-smith** [comment]
> Is there any reason that tools like `sudo`, `ip`, and `brctl` are accessed indirectly (through shell variables containing hardcoded absolute paths)

I don't know this specific authors intent, but that's often done as a prophylactic security measure, to prevent an attacker-crafted PATH causing the script to invoke an rogue utility.  But, I think that goes back to the days of set-uid scripts, and might not even be relevant anymore.

File: ansible/setup-offline-sources.yml [hunk 1]
```diff
@@ -66,7 +66,10 @@
 # If your offline repo's debian key has expired, uncomment this block.
 #############
 # If you had to uncomment this block, comment out the docker-ce repo logic in
+# that starts at line 56 of
```
Comment (stephen-smith) [inline ansible/setup-offline-sources.yml:69]: "in that starts" ?  I think the "in" needs to be removed with the rewording.

File: offline/kvm-hetzner.md [hunk 2]
```diff
@@ -80,11 +85,12 @@ tar -xzf ../wire-server-deploy-static-*.tgz
 tar -xf debs.tar
 ```
 
-### Install Docker from debian archive.
+### (FIXME: add iptables) Install Docker from debian archive.
```
Comment (stephen-smith) [inline offline/kvm-hetzner.md:88]: I think you can drop the FIXME since you did add the `apt install iptables` line below.

**Review by stephen-smith — APPROVED** [review]

**Comment by julialongtin** [comment]
closing. we've passed this.

**Pull request closed without merging**